src/main/java/io/pravega/connectors/hadoop/Helper.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
```

Comment:
2019

Reply:
I think we need to fix the copyright on all the files and I suggest that we take it up as a separate issue

Reply:
I think we are supposed to use the year of first publication as the copyright year (see https://en.m.wikipedia.org/wiki/Copyright_notice). Since this file is new 2019 makes sense to me for this class, but not for everything else (as per this: https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated. I don't know for sure though. We should probably get this clarified.

Reply:
fixed